### PR TITLE
Add some more badges

### PR DESCRIPTION
**Pull request by fatmcgav**
Because everyone loves badges...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,6 +3,9 @@
 [![Puppet Forge endorsed](https://img.shields.io/puppetforge/e/elastic/elasticsearch.svg)](https://forge.puppetlabs.com/elastic/elasticsearch)
 [![Puppet Forge Version](https://img.shields.io/puppetforge/v/elastic/elasticsearch.svg)](https://forge.puppetlabs.com/elastic/elasticsearch)
 [![Puppet Forge Downloads](https://img.shields.io/puppetforge/dt/elastic/elasticsearch.svg)](https://forge.puppetlabs.com/elastic/elasticsearch)
+[![Puppet Forge Score](https://img.shields.io/puppetforge/f/elastic/elastic_stack.svg)](https://forge.puppetlabs.com/elastic/elastic_stack)
+[![Build Status](https://travis-ci.org/elastic/elasticsearch.png?branch=master)](https://travis-ci.org/elastic/elasticsearch)
```
Comment (jmlrt) [inline README.md]: Is this message expected when clicking on the travis link?

Screenshot 2020-03-13 at 12 34 11

File: README.md [hunk 2]
```diff
@@ -3,6 +3,9 @@
...
+<!-- [![Code Coverage](https://coveralls.io/repos/github/elastic/elasticsearch/badge.svg?branch=master)](https://coveralls.io/github/elastic/elasticsearch) -->
```
Comment (jmlrt) [inline README.md]: Why is this link commented?

File: README.md (hunk 1 above)
Comment (fatmcgav) [reply README.md]: Ah, looks like I typo'd the link... Let me fix that :)

File: README.md (hunk 2 above)
Comment (fatmcgav) [reply README.md]: Coveralls isn't currently setup, but I plan to do it in the near future :)

**Pull request merged**